packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `8.10.2`
+
+- bugfix: fixed a typo in the syntax validation code for positional arguments. the bug caused the validation to never fail. [#2384](https://github.com/zowe/zowe-cli/pull/2384)
+- enhancement: added a regression test to validate the fix. [#2384](https://github.com/zowe/zowe-cli/pull/2384)  
+  (note: non-user-facing items like regression tests are typically not included in the changelog).
```

Comment:
Please use capitalization, link to the issue instead of the PR, and remove the enhancement since changes to tests don't need to be listed.

```suggestion
- BugFix: Fixed a typo in the syntax validation code for positional arguments which caused the validation to never fail. [#2375](https://github.com/zowe/zowe-cli/issues/2375)
```